server/channels/app/user_test.go

```diff
@@ -2292,161 +2291,3 @@ func TestGetUsersForReporting(t *testing.T) {
 		require.NotNil(t, userReports)
 	})
 }
-
-func TestCreateUserOrGuest(t *testing.T) {
```

Comment:
Is this intended to be deleted?

Reply:
It was: this function was mostly just testing the limits aspect of `CreateUserOrGuest` (and not the method holistically), so I opted to expand the coverage in the new method instead.